pkg/minishift/network/settings_windows.go

```diff
@@ -20,13 +20,14 @@ import (
 	"encoding/base64"
 	"fmt"
 	"net"
-	"os"
 	"strings"
 	"time"
 
 	hvkvp "github.com/gbraad/go-hvkvp"
 	"github.com/golang/glog"
 	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/minishift/constants"
```

Comment:
I would say stick to one format so update this as 
```
minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
```

so that next time user will not do any error by adding another import.

Reply:
the problem is that we RELY on minikube constants, while we should have moved away a long time ago.

Reply:
@anjannath I agree with budhram.